dbbs_models/golgi_cell_models.py

```diff
@@ -143,7 +143,7 @@ class GolgiCell(DbbsNeuronModel):
             or id == 84
             or id >= 105
             and id <= 150,
-            "arbor": '(distal-interval (proximal (region "dendrites")) 30)',
+            "arbor": '(distal-interval (proximal (tag 3)) 30)',
```

Comment:
I think `"dendrites"` is always added as a default region with `"(tag 3)"` as expression.